client/components/happychat/composer.jsx

```diff
@@ -88,7 +88,7 @@ export const Composer = createReactClass( {
 						value={ message }
 					/>
 				</div>
-				<button className="happychat__submit" disabled={ disabled } onClick={ this.sendMessage }>
+				<button primary className="happychat__submit" disabled={ disabled } onClick={ this.sendMessage }>
```

Comment:
Is this using the `<Button />` React component or a `<button />` HTML element? I don't see an import of the `<Button />` component at the top of the file so I'm unsure whether the `primary` attribute has any effect?

Reply:
You're right, completely missed that...do you think the commit I just added would help take care of this?

Reply:
Yep, I think this works fine 👍